src/sdltiles.cpp

```diff
@@ -369,15 +427,32 @@ bool WinCreate()
     const Uint32 wformat = SDL_GetWindowPixelFormat( ::window.get() );
     format.reset( SDL_AllocFormat( wformat ) );
     if( !format ) {
-        dbg(D_ERROR) << "SDL_AllocFormat(" << wformat << ") failed: " << SDL_GetError();
+        dbg( D_ERROR ) << "SDL_AllocFormat(" << wformat << ") failed: " << SDL_GetError();
         return false;
     }
 
-    bool software_renderer = get_option<bool>( "SOFTWARE_RENDERING" );
+    std::string riname = "opengl";
+    if( !get_option<std::string>( "RENDERER" ).empty() ) {
+        std::string riname = get_option<std::string>( "RENDERER" );
```

Comment:
This statement is kind of pointless, the variable goes out of scope directly afterwards anyway, why set it at all? (Did you mean to assign it to the already existing variable `riname`?)

Reply:
Thank! It seems I've added excessive variable declaration inside of if.